### PR TITLE
Load HRTF from memory instead of file 

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Comment by fluffy** [comment]
Hi @hoene - Would you be interested in including something like this in libmysofa. It's not really ready to merge yet but wanted to get your feedback on it. It allows the HRTF to be loaded from memory intead of a file. This makes it much easier to use the library on android and IOS particularly when using unity to develop apps targeted at mobile devices and AR/VR headsets.  The downside is I don't think that windows supports the funopen function as the OS level which this relies on. 

Great to hear your thoughts - thanks Cullen. And thanks for excellent HRTF library.

File: CMakeLists.txt [hunk 1]
```diff
@@ -5,8 +5,8 @@ include(CheckCCompilerFlag)
 include(GenerateExportHeader)
 include(GNUInstallDirs)
 
-option(BUILD_TESTS "Build test programs" ON)
-option(BUILD_SHARED_LIBS "Build shared library" ON)
+option(BUILD_TESTS "Build test programs" OFF)
+option(BUILD_SHARED_LIBS "Build shared library" OFF)
```
Comment (hoene) [inline CMakeLists.txt:9]: I prefer to have these changes.

**Comment by hoene** [comment]
Looks for. Actually, embedded vr headset was the use case, which I had in mind for libmysofa.

File: CMakeLists.txt (hunk 1 above)
Comment (fluffy) [reply CMakeLists.txt:9]: Ah, yes, this was just a mistake on my part to add a local cmake file in. I will remove that

**Comment by fluffy** [comment]
This has been replaced with new PR#184

**Pull request closed without merging**